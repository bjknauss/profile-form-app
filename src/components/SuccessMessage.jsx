import React from 'react'
import withMessage from './MessageWrapper'

let SuccessMessage = ({message}) => (
  <div className="alert alert-success">
    {message}
  </div>
)

SuccessMessage = withMessage(SuccessMessage)

export default SuccessMessage