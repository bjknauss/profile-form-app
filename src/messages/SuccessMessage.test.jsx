import React from 'react'
import renderer from 'react-test-renderer'
import SuccessMessage from './SuccessMessage'

test('Displays with success message', () => {
  const successMessage = "test success message"
  const component = renderer.create(
    <SuccessMessage message={successMessage} />
  )

  let tree = component.toJSON()
  expect(tree).toMatchSnapshot()
})