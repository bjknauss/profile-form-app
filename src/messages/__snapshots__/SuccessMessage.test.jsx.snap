// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Displays with success message 1`] = `
<div
  className="alert alert-success"
>
  test success message
</div>
`;
